storage/object/api-cli/bucket-policy.mdx

```diff
@@ -165,50 +173,54 @@ Bucket policies use a JSON-based access policy language and are composed of stri
 
 ## Sid
 
-**Description** 
+**Description**
 : Provides a way to include information about an individual statement.
 
-**Required** 
+**Required**
 : No
 
-**Parent** 
+**Parent**
 : Statement
 
-**Type** 
+**Type**
 : string
 
-**Sample** 
+**Sample**
 
 ```json
 {
     "Version": "2012-10-17",
     "Id":"MyBucketPolicy",
     "Statement": [
         {
-            "Sid": "Grant list to <PROJECT_ID>",
+            "Sid": "Grant list to user",
             "Effect":"Allow",
             "Principal":{
-                "SCW":"project_id:<PROJECT_ID>"
+                "SCW":"user_id:<USER_ID>"
             },
             "Action":"s3:ListBucket",
             "Resource":"<BUCKET_NAME>"
         }
     ]
 }
-``` 
+```
 
 ## Principal
 
-**Description** 
-: Provides a way to specify a principal using the ProjectID of a Scaleway Project. Note that you can use `*` to grant access to "everyone".
+**Description**
+: Provides a way to specify principals, eg. targets of the bucket policies, which can be a [user](/identity-and-access-management/iam/concepts#user) or an [application](/identity-and-access-management/iam/concepts#application). You must use the `user_id` and/or `application_id`, to target these. Note that you can use `*` to grant access to "everyone".
```

Comment:
We need to discuss about how to talk about the former system of projects as principals. This could be explained here, among other places

Reply:
I've added a message box to talk about this in the "combining" page, do you think it's enough?